public/ifcWorker.js

```
@@ -496,6 +500,10 @@ self.onmessage = async (event) => {
         await handleSqliteQuery({ ...data, messageId });
         break;
 
+      case "exportIfc":
+        await handleExportIfc({ ...data, messageId });
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)